### PR TITLE
Revert "[EMCAL-728] Reduce verbosity of the error logging"

### PR DESCRIPTION
**Pull request by mfasDa**
- Revert "[EMCAL-728] Reduce verbosity of the error logging"
- Suppress gain-related errors after the max. amount of messages is reached
- Reset error counters after 10 minutes

### CODE REVIEW

File: Detectors/EMCAL/workflow/src/RawToCellConverterSpec.cxx [hunk 1]
```diff
@@ -510,6 +501,5 @@ o2 ::framework::DataProcessorSpec o2::emcal::reco_workflow::getRawToCellConverte
                                           o2::framework::Options{
                                             {"fitmethod", o2::framework::VariantType::String, "gamma2", {"Fit method (standard or gamma2)"}},
                                             {"maxmessage", o2::framework::VariantType::Int, 100, {"Max. amout of error messages to be displayed"}},
-                                            {"printtrailer", o2::framework::VariantType::Bool, false, {"Print RCU trailer (for debugging)"}},
-                                            {"mergeHGLG", o2::framework::VariantType::Bool, true, {"Merge HG and LG channels for same tower"}}}};
```
Comment (davidrohr) [inline Detectors/EMCAL/workflow/src/RawToCellConverterSpec.cxx]: could you please only revert the logging, not the mergeHGLG?

Comment (mfasDa) [reply Detectors/EMCAL/workflow/src/RawToCellConverterSpec.cxx]: Of course, sorry!

**Comment by mfasDa** [comment]
@davidrohr nomerge mode is re-established.

**Comment by mfasDa** [comment]
CS8 error seems to be unrelated

**Comment by davidrohr** [comment]
ok, will try this later and merge it for the next nightly

**Review by davidrohr — APPROVED** [review]
ok for now, we have to see whether 100 messages per processes is not too much...
this can still lead to severel 100k messages per hour.

**Comment by davidrohr** [comment]
had to squash-merge, since rebase-merge didn't work cleanly.
But this should go into the nightly so EMCAL can use it tomorrow.

**Pull request merged**